src/main/java/minicraft/screen/SkinDisplay.java

```diff
@@ -149,14 +148,53 @@ public void render(Screen screen) {
 		Font.drawCentered("Use "+ Game.input.getMapping("cursor-down") + ", " + Game.input.getMapping("cursor-up") + ", " + Game.input.getMapping("SELECT"), screen, Screen.h - 11, Color.get(0, 222, 222, 222)); // Controls
 
 		int h = 2;
-		int w = 15;
+		int w = 16;
 		int xo = (Screen.w - w * 8) / 2;
 		int yo = 28;
 
-		// TODO: Replace the logo with a preview of the skin to be selected
 		for (int y = 0; y < h; y++) {
 			for (int x = 0; x < w; x++) {
-				screen.render(xo + x * 8, yo + y * 8, x + y * 32, 0, 3); // Skin preview?
+				if (selected == 0) {
```

Comment:
Find out a way to make this more dynamic. Start by making this a for loop.

Reply:
You mean to simplify it so it's smaller?

Reply:
Exactly!